### ISSUE TITLE
Feature request: insert recipes manually

### ISSUE THREAD

**Issue by Haridasi** [Chobbes/org-chef#18]
Is there a way to insert recipes without an url?

**Comment by Chobbes** [comment]
Currently no! How would you want such a feature to look? I just manually type such entries in, and I'm not sure how to make that much more convenient.

**Comment by Haridasi** [comment]
My suggestion is a new capture template that asks for title and then fills this out, ready to be filled at my convenience :)
* My Homemade cooking recipe title
  :PROPERTIES:
  :source-url: 
  :servings: 
  :prep-time: 
  :cook-time: 
  :ready-in: 
  :END:
** Ingredients

   - 

** Directions

   1.

**Comment by Haridasi** [comment]
I have only about 6 months of experience with emacs, but my impression is that there really are only two options. Either some manual init.el editing or it has to be done using keyboard shortcuts as input methods.

**Comment by Chobbes** [comment]
Yeah, I think I could provide the capture templates in variables or something, but I think people would likely modify them anyway? Doesn't save too much, but maybe I should make an issue of it.

**Comment by Chobbes** [comment]
Something like this should tide you over?

          ("m" "Manual Cookbook" entry (file "~/org/cookbook.org")
           "* %^{Recipe title: }\n  :PROPERTIES:\n  :source-url:\n  :servings:\n  :prep-time:\n  :cook-time:\n  :ready-in:\n  :END:\n** Ingredients\n   %?\n** Directions\n\n")

**Comment by Chobbes** [comment]
README is updated with it, hope that helps!

**Comment by Haridasi** [comment]
Just got a chance to test it. Thank you! Works wonderfully and I'm grateful for the feature added :)
